COPYRIGHT.txt

```diff
@@ -173,6 +173,26 @@ Comment: The FreeType Project
 Copyright: 1996-2017, David Turner, Robert Wilhelm, and Werner Lemberg.
 License: FTL
 
+Files: ./thirdparty/harfbuzz/
+Comment: HarfBuzz
+Copyright: 2010, 2011, 2012, Google, Inc.
+ 2012, Mozilla Foundation
+ 2011, Codethink Limited
+ 2008, 2010, Nokia Corporation and/or its subsidiary(-ies)
+ 2009, Keith Stribley
+ 2009, Martin Hosken and SIL International
+ 2007, Chris Wilson
+ 2006, Behdad Esfahbod
+ 2005, David Turner
+ 2004, 2007, 2008, 2009, 2010, Red Hat, Inc.
+ 1998-2004, David Turner and Werner Lemberg
+License: MIT
```

Comment:
The text of Harfbuzz's MIT variant should be included, as it's not the same as the "classical" MIT license Godot uses, or as Debian names it the Expat license.

Since there are dozens of MIT licenses with different wordings, I'd suggest to name this one `MIT-HarfBuzz` for clarity.

For the reference, I checked https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-specification and https://spdx.org/licenses/ but it doesn't include any identifier for HarfBuzz's MIT variant.
Debian itself documents it as "MIT" in https://metadata.ftp-master.debian.org/changelogs/main/h/harfbuzz/harfbuzz_1.8.8-2_copyright, but since we might end up adding more thirdparty code under other MIT variants, I prefer to be more explicit.